charts/scalardb-analytics-postgresql/values.yaml

```
@@ -33,31 +33,31 @@ scalardbAnalyticsPostgreSQL:
     # Storage implementation. "cassandra" or "cosmos" or "dynamo" or "jdbc" or "grpc" can be set. Default storage is "cassandra".
     scalar.db.storage=${env:SCALAR_DB_STORAGE:-}
 
-  # -- Secret name that includes sensitive data such as credentials. Each secret key is passed to Pod as environment variables using envFrom.
+  # -- Secret name that includes sensitive data such as credentials. Each secret key is passed to a pod as environment variables by using envFrom.
   secretName: ""
 
   image:
     # -- Docker image repository of ScalarDB Analytics with PostgreSQL.
     repository: ghcr.io/scalar-labs/scalardb-analytics-postgresql
-    # -- Specify a image pulling policy.
+    # -- Specify an image-pulling policy.
     pullPolicy: IfNotPresent
-    # -- Override the image tag whose default is the chart appVersion
+    # -- Override the image tag with a default that is the chart appVersion.
     tag: ""
 
   # -- Optionally specify an array of imagePullSecrets. Secrets must be manually created in the namespace.
   imagePullSecrets: [name: reg-docker-secrets]
 
   strategy:
     rollingUpdate:
-      # -- The number of pods that can be created above the desired amount of pods during an update
+      # -- The number of pods that can be created above the desired amount of pods during an update.
       maxSurge: 25%
-      # -- The number of pods that can be unavailable during the update process
+      # -- The number of pods that can be unavailable during the update process.
       maxUnavailable: 25%
     # -- New pods are added gradually, and old pods are terminated gradually, e.g: Recreate or RollingUpdate
     type: RollingUpdate
 
   service:
-    # -- service types in kubernetes
+    # -- Service types in kubernetes
```

Sorry, I missed the capitalization of `Kubernetes` here.

```suggestion
    # -- Service types in Kubernetes
```

---

Oh, I also overlooked it. Thank you! 🙇‍♀️